host/sgx/loadelf.c

```diff
@@ -535,30 +535,29 @@ static uint64_t _make_secinfo_flags(uint32_t flags)
 
 static oe_result_t _add_relocation_pages(
     oe_sgx_load_context_t* context,
-    uint64_t enclave_addr,
-    const void* reloc_data,
-    const size_t reloc_size,
+    uint64_t enclave_base,
+    oe_enclave_elf_image_t* image,
```

Comment:
```suggestion
    const oe_enclave_elf_image_t* image,
```

Reply:
Just checked, this can be const. Updated PR. @CodeMonkeyLeet could you double check the updated PR?